### PR TITLE
refactor(client): add job manifest model

### PR DESCRIPTION
**Pull request by jialeicui**
## Modules
- [ ] UI
- [ ] Controller
- [ ] Agent
- [x] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/utils/__init__.py [hunk 1]
```diff
@@ -148,7 +149,10 @@ def get_downloadable_sw_version() -> str:
 
 def get_field(item: t.Dict, field: str):  # type:ignore
```
Comment (goldenxinxing) [inline client/starwhale/utils/__init__.py:150]: Is it mean that the type of item is Any?

Comment (jialeicui) [reply client/starwhale/utils/__init__.py]: Fixed

**Review by goldenxinxing — DISMISSED** [review]
LGTM

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**